Dockerfile

```diff
@@ -0,0 +1,82 @@
+##################################
+# Build
+##################################
+FROM node:14 as build
```

Comment:
Curious: what does the `as build` syntax do here? I have some familiarity with Docker but haven't seen this before.

Reply:
It is for multi-stage builds. See: https://docs.docker.com/develop/develop-images/multistage-build/

These are useful for controlling the number of layers in the final image and also for controlling the contents of the image itself. Most of the time, the tools necessary at build time are not necessary at runtime. Using a multi-stage build allows us to leave the compiler toolchains behind and not include them in the final image.

Reply:
Ah,  I suspected as much. Thanks for the speedy response!